test/e2e/specs/editor/various/preview.spec.js

```diff
@@ -184,6 +184,20 @@ test.describe( 'Preview', () => {
 		// FIXME: The confirmation dialog is not named yet.
 		await page.click( 'role=dialog >> role=button[name="OK"i]' );
 
+		// Wait for the status change.
+		await expect(
+			page.locator( 'role=button[name="Publish"i]' )
+		).toBeVisible();
+
+		const previewToggle = page.locator(
+			'role=button[name="View"i][expanded=true]'
+		);
+
+		// Close "View" dropdown if open.
+		if ( await previewToggle.isVisible() ) {
```

Comment:
I think the `previewToggle` will always be visible here unless we add `[include-hidden]` to the locator selector?

Reply:
Maybe we should assert expanded status. We only want to close this if it's expanded.

Reply:
I think waiting for the status change might be enough in this case. Let's remove this visible check and merge this PR to test it out?

Reply:
Sounds good. Updated in ea84e045fdf25f5971a58f7be9b48e08fcd0007b.